utils/lib/Quickstart.ts

```diff
@@ -125,9 +125,11 @@ class Quickstart {
 
   /**
    * Get mutation variables from quickstart config
-   * @returns - mutation varaibles for quickstart
+   * @returns - Promised mutation variables for quickstart
    */
-  getMutationVariables(dryRun: boolean): QuickstartMutationVariable {
+  async getMutationVariables(
```

Comment:
We'll want to make sure that is function is now being awaited wherever it's used throughout the scripts

Reply:
Weird, i must've restored at some point and lost the changes i made. I'll go back and fix this!

Reply:
Only needed the fix inside `create-quickstarts-snapshot.ts` 👍🏻 . `create_validate_pr_quickstart` has the function calls mapped inside a `Promise.all`